src/it/setup-it/verify.groovy

```diff
@@ -0,0 +1,23 @@
+/*
+ *   Copyright 2016 Kamesh Sampath
```

Comment:
@kameshsampath btw, you can configure the license header in the IDEA preferences (search for 'copyright') and you can update all files, too.